summarize-text/summarize.py

```diff
@@ -12,17 +14,35 @@ def func_summarize(text, minLength, maxLength, model_name):
         model_name: The name of the model to be used for summarization.
     Returns:
         summary: The summarized text.
-    '''
+        '''
     
-    try:
-        output = pipeline('summarization', model=model_name, min_length=minLength, max_length=maxLength)
-        summary = output(text)[0]['summary_text']
-    except Exception as e:
-        return e
-    return summary
+    text = flags.text
+    min_len = flags.min
+    max_len = flags.max
+    model = flags.model
+    file = flags.file
+    
+    if not text and not file:
```

Comment:
Rather than assessing the truthiness of vars 'text' and 'files', since they will be either 'None' or something, check to see if each var == None explicitly

Reply:
Made requested change